micromanager_gui/multid_widget.py

```diff
@@ -117,6 +153,45 @@ def _set_enabled(self, enabled: bool):
         self.stage_pos_groupBox.setEnabled(enabled)
         self.acquisition_order_comboBox.setEnabled(enabled)
 
+    def _set_top(self):
+        self.z_top_doubleSpinBox.setValue(self._mmc.getZPosition())
+
+    def _set_bottom(self):
+        self.z_bottom_doubleSpinBox.setValue(self._mmc.getZPosition())
+
+    def _update_topbottom_range(self):
+        self.z_range_topbottom_doubleSpinBox.setValue(
+            abs(self.z_top_doubleSpinBox.value() - self.z_bottom_doubleSpinBox.value())
+        )
+
+    def _update_rangearound_label(self):
+        up_down_range = self.zrange_spinBox.value() / 2
```

Comment:
this one could take `value` as the first function parameter, and no longer check `self.zrange_spinBox.value()`